pwiz_tools/Skyline/SkylineFiles.cs

```diff
@@ -1950,14 +1950,14 @@ public void ImportMassList(string fileName)
                 // Try importing that list to check for errors
                 docCurrent = docCurrent.ImportMassList(input, importer, null,
                     insertPath, out selectPath, out irtPeptides,
-                    out librarySpectra, out testErrorList, out peptideGroups);
+                    out librarySpectra, out testErrorList, out peptideGroups, null, SrmDocument.DOCUMENT_TYPE.none, hasHeaders);
                 if (!testErrorList.Any())
                 {
                     useColSelectDlg = false;
                 }
             }
 
-            if (useColSelectDlg)
+            if (useColSelectDlg || forceDlg)

```

Comment:
I think at this point the code could be simplified by ORing useColSelectDlg and forceDlg?